scripts/cpplinter.sh

```diff
@@ -24,6 +24,7 @@ EXCLUDED_PATTERNS="\
     /_build \
     .pb. \
     /node_modules \
+    common/crypto/pdo
```

Comment:
```suggestion
    common/crypto/pdo \
```
i guess in this particular case it does work without but for general robustness i think the \ would be good to have ...